.travis.yml

```diff
@@ -68,13 +68,10 @@ addons:
 language: cpp
 compiler: clang
 
-before_install:
-    - curl --retry 10 --retry-max-time 120 -L "https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2" | tar xfj -
-
 install:
-  - export PATH=$PATH:$PWD/gcc-arm-none-eabi-5_4-2016q2/bin
+  - make arm_sdk_install
 
-before_script: arm-none-eabi-gcc --version
+before_script: tools/gcc-arm-none-*/bin/arm-none-eabi-gcc --version
```

Comment:
What if I have muliple versions installed? Not sure a wildcard in the path is a good idea.